american-journal-of-medical-genetics.csl

```diff
@@ -24,13 +24,13 @@
   </info>
   <macro name="editor">
     <names variable="editor" delimiter=", ">
-      <name name-as-sort-order="all" sort-separator=", " initialize-with="" delimiter=", " delimiter-precedes-last="always"/>
+      <name name-as-sort-order="all" sort-separator=", " initialize-with="" delimiter=" " delimiter-precedes-last="always"/>
```

Comment:
Fixed the delimited and sort-separator attributes. 

Here's what that link says in the "References" section:
```
_____ References are cited in text with brackets, not parentheses.
_____ References in bibliography are alphabetized, without numbering.
_____ Every author is specified.
_____ Authors’ name punctuation is limited to commas between authors.
_____ Page numbers include beginning and ending page number (e.g., do not use the 1083-9 convention).
```